R/get_intercom_data.R

```diff
@@ -1,13 +1,46 @@
 #' Get data from intercom API
-#' @param app_id App ID
-#' @param key API Key
+#' @param endpoint Object to access under https://api.intercom.io/
```

Comment:
Please add comments on possible values, or example if it is too many on this "endpoint".